### PR TITLE
Fix ipv4/localgw/NonHA CI

### PR DESCRIPTION
**Pull request by astoycos**
Currenlty in CI our ipv4, localgw, NonHA control-plane
tests aren't running because of a small error

```
Ran 0 of 54 Specs in 0.001 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Flaked | 0 Pending | 54 Skipped
--- PASS: TestE2e (0.00s)
PASS
ok  	github.com/ovn-org/ovn-kubernetes/test/e2e	0.045s
```

This fixes it

Signed-off-by: astoycos <astoycos@redhat.com>

### CODE REVIEW

**Comment by astoycos** [comment]
We probably need to get CI green for LGW before merging this though... It looks like some netflow tests are failing

**Comment by astoycos** [comment]
The test `e2e br-int NetFlow export validation` is failing because it references 172.18.0.5 Which is a node IP only seen in HA deployments I am going to skip this test for localGW non HA

**Comment by trozet** [comment]
@astoycos looks like there is still some issue:

```
2021-06-01T21:31:55.9824970Z   /home/runner/work/ovn-kubernetes/ovn-kubernetes/test/e2e/e2e.go:2529
2021-06-01T21:31:55.9826039Z Jun  1 21:22:37.220: FAIL: should get ovnkube-db deployment
2021-06-01T21:31:55.9826615Z Unexpected error:
2021-06-01T21:31:55.9827172Z     <*errors.StatusError | 0xc0006545a0>: {
2021-06-01T21:31:55.9827714Z         ErrStatus: {
2021-06-01T21:31:55.9828227Z             TypeMeta: {Kind: "", APIVersion: ""},
2021-06-01T21:31:55.9828742Z             ListMeta: {
2021-06-01T21:31:55.9829163Z                 SelfLink: "",
2021-06-01T21:31:55.9829676Z                 ResourceVersion: "",
2021-06-01T21:31:55.9830168Z                 Continue: "",
2021-06-01T21:31:55.9830712Z                 RemainingItemCount: nil,
2021-06-01T21:31:55.9831190Z             },
2021-06-01T21:31:55.9831597Z             Status: "Failure",
2021-06-01T21:31:55.9833041Z             Message: "deployments.apps \"ovnkube-db\" not found",
2021-06-01T21:31:55.9833894Z             Reason: "NotFound",
2021-06-01T21:31:55.9834995Z             Details: {Name: "ovnkube-db", Group: "apps", Kind: "deployments", UID: "", Causes: nil, RetryAfterSeconds: 0},
2021-06-01T21:31:55.9835768Z             Code: 404,
2021-06-01T21:31:55.9836131Z         },
2021-06-01T21:31:55.9836441Z     }
2021-06-01T21:31:55.9837121Z     deployments.apps "ovnkube-db" not found
2021-06-01T21:31:55.9837655Z occurred
```

**Comment by aojea** [comment]
it passed on the noha and fails in the ha, it was the opposite before

doesn't have a deployment the HA model?

**Comment by astoycos** [comment]
Yep I had to hop off late this afternoon, I'll fix that tonight, thanks for the heads up!

File: test/scripts/e2e-cp.sh [hunk 1]
```diff
@@ -12,13 +12,20 @@ if [ "$KIND_IPV4_SUPPORT" == true ] && [ "$KIND_IPV6_SUPPORT" == true ]; then
...
-
 if [ "$OVN_HA" == false ]; then
   if [ "$SKIPPED_TESTS" != "" ]; then
   	SKIPPED_TESTS+="|"
   fi
   # No support for these features in no-ha mode yet
-  SKIPPED_TESTS+="recovering from deleting db files while maintain connectivity"
+  # TODO streamline the db delete tests
+  SKIPPED_TESTS+="recovering from deleting db files while maintain connectivity|\
+Should validate connectivity before and after deleting all the db-pods at once in HA mod"
```
Comment (trozet) [inline test/scripts/e2e-cp.sh:22]: s/mod/mode?

Comment (astoycos) [reply test/scripts/e2e-cp.sh]: Fixed thanks!

**Review by trozet — APPROVED** [review]
thanks for fixing this @astoycos !

**Pull request merged**